base/remote/R/check_qsub_status.R

```diff
@@ -25,7 +26,9 @@ qsub_run_finished <- function(run, host, qstat) {
   }
 
   if (length(out) > 0 && substring(out, nchar(out) - 3) == "DONE") {
-    PEcAn.logger::logger.debug("Job", run, "for run", run_id_string, "finished")
+    if (verbose) {
+      PEcAn.logger::logger.debug("Job", run, "for run", run_id_string, "finished")
```

Comment:
PEcAn.logger already has built-in verbosity control via `logger.setLevel()` -- is a function-specific verbose flag needed here or is it enough for the user to set the logger level to something higher than debug so that this message isn't printed?

Reply:
1) I am unsure if I fully understand how `logger.setLevel()` works inside the `PEcAn.logger` package.
2) For a small amount of jobs, I think it will still be helpful to have job info printed instead of creating a progress bar.